docs/UserGuide.adoc

```diff
@@ -165,16 +165,24 @@ Displays the list of spending tagged with `groceries` in `descending order by da
 
 === Find by keyword : `find`
 
-Searches for spending based on given keywords, cost range and date range. +
+Searches for spending based on given keywords, cost range, date range, remarks or tags. +
 At least one search field must be present. +
```

Comment:
Put these lines in a useful information legend. [User Guide Legends](https://deploy-preview-85--ay1920s1-cs2103t-f13-3.netlify.com/userguide#how-to-use-this-guide)

Would be good if could also explain in more detail on what does searching in cost/date range and name/remark keywords mean to improve readability.